src/components/organisms/UserCard/index.js

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { Button, Img, Avatar, Text } from "../../atoms";
+import themes from "../../../theme/theme";
```

Comment:
```suggestion
import React from "react";

import styled from "styled-components";
import PropTypes from "prop-types";

import themes from "../../../theme/theme";
import { Button, Img, Avatar, Text } from "../../atoms";
```

이렇게 해야 warning이 없을 것 같은데 확인 부탁드립니다🙇‍♀️